config/default/core.extension.yml

```diff
@@ -26,6 +26,7 @@ module:
   chosen: 0
   chosen_lib: 0
   ckeditor: 0
+  ckeditor5: 0
```

Comment:
Ensure [ckeditor5_plugins](https://github.com/SU-SWS/ckeditor5_plugins/) is added an enabled to avoid pasting garbage html into the wysiwygs.

Reply:
Done.